packages.md

```diff
@@ -35,6 +35,101 @@ Or you can do it with [`LOOP`][loop].
   collect s)
 ~~~
 
+# Package nickname
+
+## Nickname Provided by Packages
+
+When defining a package, it is normal to give it a nickname for better user
```

Comment:
I would note that that overuse of package aliases can wind up extremely confusing for future maintainers.

Reply:
As a man who has a bunch of nicknames, I would say "it depends".

If the nickname are meaningful and straightforward, I think it is OK. Also TBH I seldom see a package with more than one nickname.

But you are right. Giving too many nicknames to a package may lead to chaos especially when people use different nicknames simultaneously.

I will add some explanation of this. Thanks for your advice.